src/196AA0.c

```diff
@@ -287,9 +287,26 @@ s32 check_block_input(s32 buttonMask) {
     return block;
 }
 
-INCLUDE_ASM(void, "196AA0", func_80269118);
+void func_80269118(void) {
+    PlayerData* playerData = &gPlayerData;
+    ActionCommandStatus *actionCommandStatus = &gActionCommandStatus;
+
+    if (!actionCommandStatus->autoSucceed) {
+        if (playerData->otherHitsTaken < 9999U) {
```

Comment:
Do you need the U here?

Reply:
Looks like you don't if `otherHitsTaken` is switched to `u16` instead of `s16`. The other stat counters are also `u16`, that's probably the right change to make